js/components/reuse/card.vue

```diff
@@ -64,7 +64,18 @@ import placeholders from 'helpers/placeholders';
 import moment from 'moment';
 import config from 'config';
 
+const MASK = [
```

Comment:
Different day, different mood, different solution :)
Which one do you prefer: single line or joined array ?